### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by pafff**
Corrections to models and views

@bitrut

### CODE REVIEW

File: polls/views.py [hunk 1]
```diff
@@ -11,18 +12,35 @@ class IndexView(generic.ListView):
     context_object_name = 'latest_question_list'
 
     def get_queryset(self):
-        return Question.objects.order_by('-pub_date')[:5]
+        return Question.objects.filter(pub_date__lte=timezone.now()).exclude(
+            choice__isnull=True).order_by('-pub_date').distinct()[:5]
```
Comment (bitrut) [inline polls/views.py:16]: Why do you have `exclude(choice__isnull=True)` while below you it's `exclude(choice=None)`?

Comment (pafff) [inline polls/views.py:16]: I was looking for a way to exclude questions without choices, tried these two solutions and forgot to unify them. Is there a preferred way to do this?

Comment (bitrut) [inline polls/views.py:16]: `choice__isnull=True` seems to be more explicit. I would use this one.

**Comment by pafff** [comment]
Updated `get_queryset` to use `choice__isnull=True` instead of `choice=None`
@bitrut

**Comment by bitrut** [comment]
:+1:

**Pull request merged**